flutter/lib/src/sentry_privacy_options.dart

```diff
@@ -50,12 +51,39 @@ class SentryPrivacyOptions {
       assert(!maskAssetImages,
           "maskAssetImages can't be true if maskAllImages is false");
     }
+
     if (maskAllText) {
       rules.add(
           const SentryMaskingConstantRule<Text>(SentryMaskingDecision.mask));
       rules.add(const SentryMaskingConstantRule<EditableText>(
           SentryMaskingDecision.mask));
     }
+
+    // In Debug mode, check if users explicitly mask (or unmask) widgets that
+    // look like they should be masked, e.g. Videos, WebViews, etc.
+    if (kDebugMode) {
```

Comment:
wdyt of using `options.platformChecker.isDebugMode()`? 

but doesn't really make a difference I think

only matters if we add an additional test case where we assert that the warning isn't added for release builds, then we can mock it

Reply:
Thanks, I've made the changes and added test cases for non-debug builds